apps/base/utils.py

```diff
@@ -11,3 +15,20 @@ def paginated_queryset(queryset, request):
     paginator.page_size = settings.REST_FRAMEWORK['PAGE_SIZE']
     result_page = paginator.paginate_queryset(queryset, request)
     return (paginator, result_page)
+
+
+@deconstructible
```

Comment:
@TroJan I am using this decorator because it adds logic to capture and preserve the arguments on their way into our constructor, and then returns those arguments exactly.